com/course/zenkouv/src/main/java/lecture6/gitHub/GitHub.java

```diff
@@ -0,0 +1,34 @@
+package lecture6.gitHub;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lecture6.IRequestExecutor;
+import lecture6.constant.GitHubApiConstants;
+import lecture6.constant.RequestType;
+import lecture6.gitHub.controller.GitHubController;
+import lecture6.model.GitHubUser;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.List;
+
+public class GitHub {
+
+    private GitHub() {
+    }
+
+    public static void sendCollaboratorInvite(String name, IRequestExecutor requestExecutor) throws IOException, URISyntaxException {
+        new GitHubController(requestExecutor).request(GitHubApiConstants.BASE_URL +
+                GitHubApiConstants.URL_COLLABORATOR + name, RequestType.PUT);
+    }
+
+    public static String getCollaborators(IRequestExecutor requestExecutor) throws IOException, URISyntaxException {
+        return new GitHubController(requestExecutor).request(GitHubApiConstants.BASE_URL +
+                GitHubApiConstants.URL_COLLABORATOR, RequestType.GET);
+    }
+
+    public static List<GitHubUser> castResponseToContributorList(String content) throws IOException {
+        return new ObjectMapper().readValue(content, new TypeReference<List<GitHubUser>>() {
```

Comment:
move to ParseHelper as static method

Reply:
<T> T parse (String content, Class class)

Reply:
fix